### PR TITLE
DASH-3018 [Frontend] Create Device card under user

### PR DESCRIPTION
**Pull request by sinisa-nogic**
https://webcomputing.jira.com/browse/DASH-3018

### CODE REVIEW

File: src/components/account/company/appsco-account-devices-item.html [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+<link rel="import" href="../../../../bower_components/neon-animation/neon-animation-runner-behavior.html">
+<link rel="import" href="../../../../bower_components/neon-animation/animations/fade-in-animation.html">
+<link rel="import" href="../../../../bower_components/neon-animation/animations/fade-out-animation.html">
+<link rel="import" href="../../../../bower_components/paper-styles/typography.html">
+<link rel="import" href="../../../../bower_components/iron-ajax/iron-ajax.html">
+
+
+<dom-module id="appsco-account-devices-item">
+    <template>
+        <style include="appsco-list-item-styles"></style>
```
Comment (NikolaVu) [inline src/components/account/company/appsco-account-devices-item.html]: Need to import appsco-list-item-styles above.

File: src/components/account/company/appsco-account-devices-item.html [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+                    type: Object,
+                    value: function () {
+                        return {};
+                    }
+                },
+
+                /**
+                 * Indicates if devices should be in preview mode rather then full detailed view.
+                 */
+                preview: {
```
Comment (NikolaVu) [inline src/components/account/company/appsco-account-devices-item.html]: This property is unused, I think.

File: src/components/account/company/appsco-account-devices-item.html [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+                    type: Boolean,
+                    value: false,
+                    reflectToAttribute: true
+                },
+
+                _approved: {
+                    type: Boolean,
+                    computed: "_computeApproved(device)"
```
Comment (NikolaVu) [inline src/components/account/company/appsco-account-devices-item.html]: Use single quote, please :)

File: src/components/account/company/appsco-account-devices-item.html [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+
+            _onApproveAction: function(event) {
+                this.device.approved = true;
+                event.stopPropagation();
+
+                /**
+                 * Fired when `Approve` action is tapped.
+                 *
+                 * @event device-approved
+                 * @param {Object} group
```
Comment (NikolaVu) [inline src/components/account/company/appsco-account-devices-item.html]: {Object} device, {Object} account

File: src/components/account/company/appsco-account-devices-item.html [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+
+            _onDisapproveAction: function(event) {
+                this.device.approved = false;
+                event.stopPropagation();
+
+                /**
+                 * Fired when `disapprove` action is tapped.
+                 *
+                 * @event device-disapproved
+                 * @param {Object} group
```
Comment (NikolaVu) [inline src/components/account/company/appsco-account-devices-item.html]: {Object} device, {Object} account

File: src/components/account/company/appsco-account-devices-item.html [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+                this.device.approved = true;
+                event.stopPropagation();
+
+                /**
+                 * Fired when `Approve` action is tapped.
+                 *
+                 * @event device-approved
+                 * @param {Object} group
+                 */
+                this.fire('device-approved', {
```
Comment (NikolaVu) [inline src/components/account/company/appsco-account-devices-item.html]: Event should be in form of action: approve-device. 
device-approved should be used when action is done.
The same for disapprove.

File: src/components/account/company/appsco-account-devices.html [hunk 7]
```diff
@@ -0,0 +1,232 @@
...
+                 * Number of devices to load.
+                 */
+                size: {
+                    type: Number
+                },
+
+                /**
+                 * Indicates if device should be in preview mode rather then full detailed view.
+                 */
+                preview: {
```
Comment (NikolaVu) [inline src/components/account/company/appsco-account-devices.html]: Is this used?

Comment (sinisa-nogic) [reply src/components/account/company/appsco-account-devices.html]: It's used in 'appsco-manage-account-components-page' component, where only device number should be displayed, not concrete devices list.

**Pull request merged**